skyportal/handlers/api/source.py

```diff
@@ -426,6 +426,7 @@ def get(self, obj_id=None):
             q = q.filter(Obj.within(other, radius))
         if start_date:
             start_date = arrow.get(start_date.strip())
+            print(start_date)
```

Comment:
```suggestion
```

Reply:
Yeah that's what I was thinking, too. But I had the feeling there would be a lot of different stuff between the two pages, but it is starting to look like the two pages would be very similar.